### DISCUSSION TITLE
Komga/Windows Server seems to be broken again

### DISCUSSION THREAD

**Discussion by likefunbutnot** [gotson/komga#1745] [Q&A]
See subject.

The currently available Windows installation binaries either do not install or do not run on either Windows Server 2019 or 2022, usually with an entertaining variety of certificate errors, even when I have pre-imported the signed certificates from the downloaded binaries per the suggestions in this previous issue: https://github.com/gotson/komga/issues/1220.

I was able to force an installation via .MSIX. At that point, I get unhelpful xC0000005 bugchecks from komga.exe in Application event logs and absolutely nothing is written to %localappdata%/komga.  That directory doesn't exist on either of my Windows systems.

I am aware that I can run Komga in a container or desktop Windows VM but for the purposes of this matter, I am letting people who care know that the installation process is once again borked on what is supposedly a supported platform for reasons that seem to relate to acceptability of security certificates. Someone may want to give this matter some attention. As an alternative, perhaps it might be useful to remove Windows Server from the list of platforms where Komga might be said to work?

**Comment by gotson** [comment]
The Windows installer is made for Desktop, not for server. The desktop application adds a tray icon to run the service and can be easily started on startup. It is not made for servers, Windows or otherwise.

You should use the jar or docker on server.
